src/include/kernel/abi.h

```diff
@@ -29,7 +29,7 @@
 
 /** \brief SOF ABI version major, minor and patch numbers */
 #define SOF_ABI_MAJOR 3
-#define SOF_ABI_MINOR 29
+#define SOF_ABI_MINOR 30
```

Comment:
Why is this changing ABI ?

Reply:
I saw warnings when commit this patch if not changing ABI, warnings list below:
btian1@btian1-linux:~/sof_zephyr_1012/sof$ git commit -s
No codespell typos will be found - file '/usr/share/codespell/dictionary.txt': No such file or directory
WARNING: Please update ABI in accordance with http://semver.org

Reply:
ok, but tin this case we can ignore since we are not changing the ABI.